TheThingsNetwork/lorawan-stack
# New header implementation

#### Summary

New header component based on the Figma designs for the console redesign.

References: https://github.com/TheThingsIndustries/product-management/issues/29

#### Changes

Added new `header-v2` and `profile-dropdown-v2` components.
- Large screen:
![image](https://github.com/TheThingsNetwork/lorawan-stack/assets/49171937/24c052cf-b09c-40ef-8c70-aadae10cab63)

- Mobile screen:
![image](https://github.com/TheThingsNetwork/lorawan-stack/assets/49171937/0f8c370d-85dc-43b5-bce4-24886c386c8f)


#### Testing

1. Start storybook on the console.
2. Check `Header V2` and `Profile Dropdown V2` components.

#### Notes for Reviewers

@ryaplots I made minor changes on the `dropdown-v2` component as well in the `useEffect` hook.

#### Checklist
